src/platform/packages/shared/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -418,16 +418,18 @@ export const getDocLinks = ({ kibanaBranch, buildFlavor }: GetDocLinkOptions): D
       migrationGuide: `${ELASTICSEARCH_DOCS}breaking-changes.html`,
     },
     siem: {
-      guide: `${SECURITY_SOLUTION_DOCS}index.html`,
-      gettingStarted: `${SECURITY_SOLUTION_DOCS}index.html`,
-      privileges: `${SECURITY_SOLUTION_DOCS}sec-requirements.html`,
-      ml: `${SECURITY_SOLUTION_DOCS}machine-learning.html`,
-      ruleChangeLog: `${SECURITY_SOLUTION_DOCS}prebuilt-rules-downloadable-updates.html`,
-      detectionsReq: `${SECURITY_SOLUTION_DOCS}detections-permissions-section.html`,
-      networkMap: `${SECURITY_SOLUTION_DOCS}conf-map-ui.html`,
-      troubleshootGaps: `${SECURITY_SOLUTION_DOCS}alerts-ui-monitor.html#troubleshoot-gaps`,
-      ruleApiOverview: `${SECURITY_SOLUTION_DOCS}rule-api-overview.html`,
-      configureAlertSuppression: `${SECURITY_SOLUTION_DOCS}alert-suppression.html#_configure_alert_suppression`,
+      guide: `${ELASTIC_DOCS}solutions/security`,
+      gettingStarted: `${ELASTIC_DOCS}solutions/security`,
+      privileges: `${ELASTIC_DOCS}solutions/security/get-started/elastic-security-requirements`,
+      ml: `${ELASTIC_DOCS}solutions/security/advanced-entity-analytics/anomaly-detection`,
+      ruleChangeLog: `https://www.elastic.co/guide/en/security/current/prebuilt-rules-downloadable-updates.html`,
```

Comment:
>      ruleChangeLog: `https://www.elastic.co/guide/en/security/current/prebuilt-rules-downloadable-updates.html`,

There doesn't seem to be a redirect or equivalent page yet, so I've left it unchanged for now.